vendor/github.com/go-openapi/errors/headers.go

```diff
@@ -74,3 +74,12 @@ func InvalidResponseFormat(value string, allowed []string) *Validation {
 		message: fmt.Sprintf(responseFormatFail, allowed),
 	}
 }
+
+// Validate error message name for aliased property
+func (e *Validation) ValidateName(name string) *Validation {
```

Comment:
can you make these changes in: https://github.com/go-openapi/errors/blob/master/headers.go
They my get lost otherwise because this is in a vendored package, which get overwritten when I update the vendored deps